src/base/global.h

```diff
@@ -26,5 +26,22 @@
  * exception statement from your version.
  */
 
+#ifndef GLOBAL_H
+#define GLOBAL_H
```

Comment:
`#pragma once` required now instead of "include guards"

Reply:
@glassez Will do. Wasn't what aware of that policy. Thanks!